examples/flutter_gallery/lib/gallery/updates.dart

```diff
@@ -41,7 +41,7 @@ class UpdaterState extends State<Updater> {
 
     final String updateUrl = await widget.updateUrlFetcher();
     if (updateUrl != null) {
-      final bool wantsUpdate = await showDialog(context: context, child: _buildDialog());
+      final bool wantsUpdate = await showDialog(context: context, builder: (BuildContext context) => _buildDialog());
```

Comment:
best way to fix this one would be to make `_buildDialog` take a `BuildContext context` argument, then you can just pass that in directly.

Reply:
Named the parameter `_` so that method would continue using the correct BuildContext